cmd/convert-config.go

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
why is there an MIT license here?

Reply:
We have that in all files where we define cli commands

Reply:
is there a reason?

Reply:
updated